doc/modules/model_evaluation.rst

```diff
@@ -1378,6 +1378,36 @@ function::
     for an example of mean squared error usage to
     evaluate gradient boosting regression.
 
+.. _mean_squared_log_error:
+
+Mean squared log error
+----------------------
+
+The :func:`mean_squared_log_error` function computes a risk metric corresponding
+to the expected value of the logarithmic squared (quadratic) error loss or loss.
```

Comment:
@amueller I fixed this one ! There was a same typo above it as well. I fixed it on the fly.